tests/test_2022_05_31_snapshot_rewards_distribution.py

```diff
@@ -0,0 +1,86 @@
+import pytest
+
+from typing import Dict
+
+from brownie import accounts, chain
+
+from scripts.vote_2022_05_31 import start_vote
+from utils.test.snapshot_helpers import dict_zip, dict_diff, assert_no_diffs, ValueChanged
+from utils.config import contracts
+from utils.node_operators import get_node_operators
+
+
+@pytest.fixture(scope="module")
+def stranger():
+    return accounts[0]
+
+@pytest.fixture(scope='module')
+def lido_oracle_report(lido):
+    lido_oracle = accounts.at(lido.getOracle(), force=True)
+    dao_voting = accounts.at(contracts.voting.address, force=True)
+
+    def report_beacon_state(steth_rebase_mult):
+        lido.setFee(0, {'from': dao_voting})
+        (_, beacon_validators, beacon_balance) = lido.getBeaconStat()
+        total_supply = lido.totalSupply()
+        total_supply_inc = (steth_rebase_mult - 1) * total_supply
+        beacon_balance += total_supply_inc
+        assert beacon_balance > 0
+        lido.handleOracleReport(beacon_validators, beacon_balance, {'from': lido_oracle})
+
+    return report_beacon_state
+
+
+def execute_vote(ldo_holder, helpers):
+    vote_id = start_vote({"from": ldo_holder}, silent=True)[0]
+    helpers.execute_vote(
+        vote_id=vote_id,
+        accounts=accounts,
+        dao_voting=contracts.voting,
+        skip_time=3 * 60 * 60 * 24,
+    )
+
+
+def make_snapshot(lido, node_operators_registry) -> Dict[str, any]:
+    node_operators = get_node_operators(node_operators_registry)
+
+    snapshot = {}
+    for node_operator in node_operators:
+        snapshot[node_operator['name']] = lido.balanceOf(node_operator['rewardAddress'])
+
+    return snapshot
+
+
+def steps(lido, node_operators_registry, lido_oracle_report) -> Dict[str, Dict[str, any]]:
+    before_rewards_distribution = make_snapshot(lido, node_operators_registry)
+
+    lido_oracle_report(steth_rebase_mult=1.01)
```

Comment:
Send 1ETH before the report to LidoELRewardsVault to break the test

Reply:
done